src/Resizer/CropResizer.php

```diff
@@ -89,9 +89,7 @@ public function getBox(MediaInterface $media, array $settings): Box
     }
 
     /**
-     * @param array<string, int> $settings
-     *
-     * @phpstan-param array{width: int, height: int} $settings
```

Comment:
Why losing this ?

Reply:
I added those phpdoc initially but not sure if they are correct:

```
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  Line   src/Resizer/CropResizer.php
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------
  83     Parameter #1 $settings of method Sonata\MediaBundle\Resizer\CropResizer::createTargetBox() expects array('width' => int, 'height' => int), array<string, mixed> given.
 ------ ------------------------------------------------------------------------------------------------------------------------------------------------------------------------
```

The thing is this array $settings is passed on a lot of places, and we are giving it array<string, mixed> on every place.

This is one of the places where the array has more than the width and height: https://github.com/sonata-project/SonataMediaBundle/blob/master/src/Thumbnail/FormatThumbnail.php#L94-L106

Maybe we should fully understand all options that could be there and create a type.

Reply:
I have pushed a new commit to fix this, please take a look to see if I did it right.

Reply:
This seems the right way to fix it to me :)